staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -183,6 +190,28 @@ func (p *restrictedProfile) Apply(pod *corev1.Pod, containerName string, target
 	return nil
 }
 
+func (p *netadminProfile) Apply(pod *corev1.Pod, containerName string, target runtime.Object) error {
+	style, err := getDebugStyle(pod, target)
+	if err != nil {
+		return fmt.Errorf("netadmin profile: %s", err)
+	}
+
+	switch style {
+	case node:
+		allowNetadminCapability(pod, containerName)
+		useHostNamespaces(pod)
+		setPrivileged(pod, containerName)
+
+	case podCopy:
+		allowNetadminCapability(pod, containerName)
+
+	case ephemeral:
+		allowNetadminCapability(pod, containerName)
+	}
```

Comment:
Looks like `allowNetadminCapability` is used in all three debugging styles, so we could shift it up to before the switch:

```suggestion
	allowNetadminCapability(pod, containerName)

	switch style {
	case node:
		useHostNamespaces(pod)
		setPrivileged(pod, containerName)

	case podCopy, ephemeral:
		// no additional modifications needed
	}
```